src/core/ts/background-scripts/background.js/telemetry/generated.spec.ts

```diff
@@ -0,0 +1,66 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { assert } from "chai";
+// import Glean from "@mozilla/glean/webext";
+import { custom } from "./generated/pings";
+import { counterTest, eventTest, stringTest } from "./generated/test";
+// import { config } from "../../../config";
+
+const testStrVal = "test";
+const pingName = "custom";
+
+const submitAndAssert = async () => {
+  const eventTestValue = await eventTest.testGetValue(pingName);
+  const counterTestValue = await counterTest.testGetValue(pingName);
+  const stringTestValue = await stringTest.testGetValue(pingName);
+  const mostRecentEventTestValue = eventTestValue.slice().pop();
+
+  console.log({
+    eventTestValue,
+    mostRecentEventTestValue,
+    counterTestValue,
+    stringTestValue,
+  });
+
+  // assert.equal(eventTestValue.length, 1);
```

Comment:
The Glean singleton is not reset after each test, so the event arrays and counters accumulate from previous tests. An attempt to use testResetGlean was once implemented to get around this, but then removed in https://github.com/mozilla-extensions/bergamot-browser-extension/pull/76/commits/b7cbd9abac3be16d3d694e834a6e93a2e2d64bd1 since it interfered with the Glean singleton used in the rest of the extension.

Reply:
Added a code comment about this

Reply:
maybe let's fix it and expect to pass? These use cases look important

Reply:
We have to wait for Glean.js 0.9+ to be able to fix it and these test assertions are not doing anything important - they are only verifying that our generated Glean code is working in some basic degree.

Reply:
let's add TODO to fix it on Glean 0.9 then

Reply:
A TODO to fix this is already added to the code at https://github.com/mozilla-extensions/bergamot-browser-extension/pull/76/files#diff-c1d2cbdde8c049dd28020cf41815b484712770fb457cb98b16d739b8059771ceR42-R44. If you want us to file an issue, we can do that as well of course.

Reply:
(Also, we don't know if it will ever be available in Glean.js)

Reply:
right, thanks